### ISSUE TITLE
JSDoc Parse Error: Cannot read property 'type' of null

### ISSUE THREAD

**Issue by victor-homyakov** [azu/jsdoc-to-assert#9]
This JsDoc annotation produces error

```js
        /**
         * @param {{SubscriptionId,Data}} data
         */
        handleData: function(data) {
```

```
Cannot read property 'type' of null
    at Expression (C:\Code\node_modules\jsdoc-to-assert\lib\tag\Expression.js:13:18)
    at C:\Code\node_modules\jsdoc-to-assert\lib\tag\RecordType.js:23:43
    at Array.map (native)
    at RecordType (C:\Code\node_modules\jsdoc-to-assert\lib\tag\RecordType.js:20:49)
    at Function.createAssertFromTag (C:\Code\node_modules\jsdoc-to-assert\lib\AssertGenerator.js:187:51)
    at createTag (C:\Code\node_modules\jsdoc-to-assert\lib\AssertGenerator.js:72:40)
    at Array.map (native)
    at Function.createAsserts (C:\Code\node_modules\jsdoc-to-assert\lib\AssertGenerator.js:74:34)
    at Function.toAsserts (C:\Code\node_modules\jsdoc-to-assert\lib\CommentConverter.js:43:50)
    at injectParameterAssert (C:\Code\node_modules\babel-plugin-jsdoc-to-assert\lib\index.js:46:51)
```

**Comment by azu** [comment]
@victor-homyakov Thanks for report.
I'll fix this issue. #10 

It will be following assertion:
```js
assert(typeof data.SubscriptionId !== 'undefined' && typeof data.Data !== 'undefined');
```

**Comment by victor-homyakov** [comment]
Maybe also first assert for `typeof data !== 'undefined'` ?

**Comment by azu** [comment]
> Maybe also first assert for typeof data !== 'undefined' ?

Yes, You're correct! 
I'll add this.

**Comment by azu** [comment]
@victor-homyakov I've released [2.7.0](https://github.com/azu/jsdoc-to-assert/releases/tag/2.7.0 "2.7.0").
It include that fixes this issue([2.6.2](https://github.com/azu/jsdoc-to-assert/releases/tag/2.6.2 "2.6.2")) and `typeof data !== 'undefined'` check.
